web/client/api/catalog/COG.js

```diff
@@ -76,6 +76,47 @@ const fromUrl = (url, signal) => {
             .catch(()=> abortError(reject));
     });
 };
+
+const getLayerConfig = (layer, image) => {
+    const crs = getProjectionFromGeoKeys(image);
+    const extent = image.getBoundingBox();
+    const isProjectionDefined = isProjectionAvailable(crs);
+    const sample = image.getSamplesPerPixel();
+    const isRGB = image.getSampleByteSize(0) === 1 && typeof get(image, 'fileDirectory.PhotometricInterpretation') !== 'undefined';
+    const { STATISTICS_MINIMUM, STATISTICS_MAXIMUM } = image.getGDALMetadata() ?? {};
```

Comment:
I suggest to store these source metadata and desume `isRGB` on demand, because conceptually this is not the indication of rgb. Also keep in mind that layer have can potentially have multiple source images.
So something like `fileDirectory.PhotometricInterpretation`  and `{samples: [ byteSize: 1 ]}`. Something like this. I'll ask @dromagnoli some info about geotiffs to clarify this.

Reply:
Replied here: 
https://github.com/geosolutions-it/MapStore2/pull/9857#issuecomment-1895513693